deploy/kubedirector/kubedirector.hpe.com_kubedirectorstatusbackups_crd.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: kubedirectorstatusbackups.kubedirector.hpe.com
+spec:
+  group: kubedirector.hpe.com
+  version: v1beta1
+  names:
+    kind: KubeDirectorStatusBackup
+    listKind: KubeDirectorStatusBackupList
+    plural: kubedirectorstatusbackups
+    singular: kubedirectorstatusbackup
+    shortNames:
+      - kdstatusbackup
+  scope: Namespaced
+  validation:
+    openAPIV3Schema:
+      type: object
+      required: [apiVersion, kind, metadata, spec]
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          type: object
+          properties:
+            statusBackup:
```

Comment:
In practice it is, in that the "object" value here is the complete kdcluster status stanza.

Ideally (arguably) the validation schema here should have all of the stuff that is in the validation schema for the status stanza. However I really don't want to manually copy all of that; that seems like a recipe for disaster when future status changes happen.

In the end since it is only KD that is writing/reading this -- and only by copying directly from and writing directly to the kdcluster status stanza  -- I'm OK with not having a validation schema here.

If we did want to have a schema here, we could for example come up with a way to define the status schema only in one document and then have the "make deploy" process build the actual CRDs used for kdcluster and kdstatusbackup. However that would introduce problems for folks who just want to directly apply the CRDs out of this repo for development/testing, upgrade, or whatever.

Reply:
sure make sense, if someone manually tries to tinker with backup they can easily put KD in panic state. Is that ok?

Reply:
In the Tenant Role/RoleBinding, we should revoke all access for everyone for kdstatusbackup (Not in this PR obviously)

Reply:
Yeah, point taken. Basically nobody but KD and the backup solution should be given privileges on kdstatusbackup CRs (one of the reasons we're using a custom CR rather than say ConfigMap).

If we did have a validation schema here it would help somewhat to protect against unexpected manual fiddling, although I think probably you could still also construct something that would upset KD.